### PR TITLE
Add flag for hasAccount to segment identify call

### PR DESCRIPTION
**Pull request by rynbyjn**
[Finishes #145880059](https://www.pivotaltracker.com/story/show/145880059)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2140,7 +2140,7 @@ elliptic@^6.0.0:
 
 "ello-brains@https://github.com/ello/brains.git":
   version "1.0.0"
-  resolved "https://github.com/ello/brains.git#7bfc95788b54e9893a93be22c5b9214a241f5061"
+  resolved "https://github.com/ello/brains.git#794028d26cb743d3a9ba55b97992088d63a2832b"
```
Comment (rynbyjn) [inline yarn.lock]: This is the SHA of brains before editorial was merged and works with the current live webapp.

Comment (jayzes) [inline yarn.lock]: Worth explicitly locking in `package.json` too, or not yet?

File: package.json [hunk 2]
```diff
@@ -119,7 +119,7 @@
     "cookie-parser": "1.4.3",
     "css-loader": "0.28.0",
     "dotenv": "4.0.0",
-    "ello-brains": "https://github.com/ello/brains.git",
+    "ello-brains": "https://github.com/ello/brains.git#794028d26cb743d3a9ba55b97992088d63a2832b",
```
Comment (mkitt) [inline package.json]: 🥇

File: src/containers/AnalyticsContainer.js [hunk 3]
```diff
@@ -13,15 +13,15 @@ import { isElloAndroid } from '../lib/jello'
 
 const agent = isElloAndroid() ? 'android' : 'webapp'
 
-export function addSegment(uid, createdAt, isFeatured, isNabaroo) {
+export function addSegment({ createdAt, hasAccount, isFeatured, isNabaroo, uid }) {
```
Comment (mkitt) [inline src/containers/AnalyticsContainer.js]: :+1:

**Pull request merged**